.github/dependabot.yml

```diff
@@ -4,3 +4,8 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+  - package-ecosystem: "gomod"
+    directory: "/.github/workflows/dapr-bot"
+    schedule:
+      interval: "daily"
+      
```

Comment:
```suggestion
```
Small nit, can we remove this line please

Reply:
Done, Changes were made